frontend/src/pages/RunDetails.test.tsx

```diff
@@ -1418,6 +1419,78 @@ describe('RunDetails', () => {
     });
   });
 
+  describe('task details tab', () => {
+    it('shows node detail info', async () => {
+      testRun.pipeline_runtime!.workflow_manifest = JSON.stringify({
+        status: {
+          nodes: {
+            node1: {
+              id: 'node1',
+              displayName: 'Task',
+              phase: 'Succeeded',
+              startedAt: '1/19/2021, 4:00:00 PM',
+              finishedAt: '1/19/2021, 4:00:02 PM',
+            },
+          },
+        },
+        metadata: { namespace: 'ns' },
+      });
+      tree = shallow(<RunDetails {...generateProps()} />);
+      await getRunSpy;
+      await TestUtils.flushPromises();
+      clickGraphNode(tree, 'node1');
+      tree
+        .find('MD2Tabs')
+        .at(1)
+        .simulate('switch', STEP_TABS.TASK_DETAIL);
+      await getRunSpy;
+      await TestUtils.flushPromises();
+
+      expect(tree.find(NODE_DETAILS_SELECTOR)).toMatchInlineSnapshot(`
```

Comment:
I mentioned enzyme to react-testing-library migration, but your test seems reasonable to stay in enzyme for now, because the existing pattern already satisfies what we need.

Reply:
Sounds good, I will explore more on react-testing-library in the future and see fits between 2 libraries when making change.